app/models/concerns/searchable.rb

```diff
@@ -20,8 +20,35 @@ def search(name_or_phone_str, lines = LINES)
       sort_and_limit_patient_matches(all_matching_names, all_matching_phones, all_matching_identifiers)
     end
 
+    def search_users(name_str)
+      name_regexp = /#{Regexp.escape(name_str)}/i
+      all_matching_names = find_name_matches_user name_regexp
+
+      sort_and_limit_user_matches(all_matching_names)
+
+    end
+
```

Comment:
I'm interested in this! I feel like in some ways User and Patient are such radically different models, where we care about much different things, that I'm okay not having them share a search method. I think that if this winds up being a big pain in the neck it would be 100% acceptable to move this into `PatientSearchable` and have another `UserSearchable` concern. 

A thing I **would** like is to continue to invoke `Patient.search` in controllers and models. So let's make sure whatever solution we implement, we're still doing that where appropriate.